esphome/config_validation.py

```diff
@@ -1171,9 +1171,7 @@ def ipv4address(value):
 def ipv4address_multi_broadcast(value):
     address = ipv4address(value)
```

Comment:
```suggestion
    return ipaddress_multi_broadcast(ipv4address(value))
```
Other than the `ipv4address` call,the code is just repeated in the other func.